gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
I think the point is that we use `GCD_VERSION` constant in LocalGcdHelper's code to check that the installed gcd version matches the one we require. If not, we download `GCD` which itself contains `GCD_VERSION` as substring. Doing as @aozarov proposes would require only changing `GCD_VERSION` and `GCD_BUILD` the day we need to update to a new gcd/API version. 

I think we have more chances of remembering to update `GCD_VERSION` and `GCD_BUILD` (as their names make sense) rather than `GCD_VERSION` and `GCD` where in `GCD` we need to change both version and build numbers.